plugins/scaffolder-backend/src/scaffolder/stages/templater/cra/index.ts

```diff
@@ -48,13 +50,33 @@ export class CreateReactAppTemplater implements TemplaterBase {
       },
     });
 
+    const extraAnnotations: Record<string, string> = {};
+    const finalDir = path.resolve(
+      resultDir,
+      options.values.component_id as string,
```

Comment:
these things always scare me a bit, better hope the id didn't contain dots and slashes... we may want to check that the result is below resultDir or something

Reply:
Good suggestion